src/app/component/ProductGallery/ProductGallery.component.js

```diff
@@ -45,16 +52,27 @@ class ProductGallery extends Component {
             }, ...Array(3).fill({})];
     }
 
+    changeActiveImage(activeImage) {
+        this.setState({ activeImage });
+    }
+
     renderAdditionalPicture({ image, type }, index = 0) {
         return (
-            <Image
-              // eslint-disable-next-line react/no-array-index-key
-              key={ index }
-              src={ image }
-              ratio="custom"
-              objectFit="cover"
-              mix={ { block: 'ProductGallery', elem: 'Image', mods: { type } } }
-            />
+            <button
+              block="ProductGallery"
+              elem="Image"
+              mods={ { type } }
+              onClick={ () => this.changeActiveImage(index) }
```

Comment:
Let's bind changeActiveImage to this in constructor. No need for arrow function here.